tgui/packages/tgui/interfaces/CharacterPreferences/index.tsx

```diff
@@ -46,7 +66,7 @@ export const CharacterPreferences = (_props: any, context: any) => {
           </Stack.Item>
           <Stack.Item grow="1">
             {menu === CharacterPreferencesTabKeys.General || menu === CharacterPreferencesTabKeys.Character ? (
-              <Stack fill>
+              <Stack fill onKeyDown={handleKeyDown}>
```

Comment:
Sanity check on if this is the correct spot for the handler. I think it's probably fine, but rotating the sprite is such a non-destructive action that I think we could get away with putting up on `Window.Content`. If a given control wants to do something else with the arrows they can still intercept.

Reply:
It currently only handles rotating the preview, so thought it would be logical to only fire those events if the preview is even rendered.

Reply:
Yes, that makes sense, but the location of the handler means that if you focus the window but not the specific `Stack` the handler is on (i.e. I load a save, which clicks outside of that area) then I don't think the arrow keys will work even though it is rendering?

You could have the `onKeyDown` be on `Window.Content` and do a tab check within `handleKeyDown`?

(Above is with the caveat that my original statement is true, feel free to verify that first.)

Reply:
Yep, moved the `onKeyDown` up to `Window.Content` and added a tab check.